doc/code/qml_debugging.rst

```diff
@@ -0,0 +1,17 @@
+qml.interfaces
+==============

```

Comment:
Wondering where the name comes from, and won't this be mixed up with the ML interfaces?

Reply:
Oh dear, that's a copy-paste error!